_posts/2020-05-06-cockpit-ansible.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Scale Cockpit troubleshooting to your computing fleet
+author: pitti
+date: 2020-05-06
+category: announcement
+tags: cockpit selinux ansible scaling troubleshooting
+comments: true
+---
+
+You know Cockpit as a troubleshooting and learning tool for an individual machine. But once you discovered and tested a solution, wouldn't it be nice to apply it to all your other machines in your data center? Of course not every problem works that way: you don't extend an LVM with a new hard disk on a hundred machines at the same time. But there *are* situations where this makes sense, for example adjusting the [SELinux policy](https://www.redhat.com/en/topics/linux/what-is-selinux) to your needs.
```

Comment:
Why the "might think of"? That sounds like it would be wrong, but it's how we want to, and are, posing it. :+1 for the other suggestions.

Reply:
"You might know Cockpit as..." could also work.

You need a _might_ or _may_  in there. People reading this might not even know about Cockpit. Or might just barely know about it.

Other options:
- "Cockpit is usually thought of as a troubleshooting..."
- "Cockpit is generally known as a troubleshooting..."

Also:
Using the phrase "learning tool" sounds like education and that you'd grow away from Cockpit. Like graduating from school.

Reply:
Thanks for the explanation!